src/java.base/share/classes/sun/security/ssl/SSLSocketImpl.java

```diff
@@ -1329,7 +1329,7 @@ public void close() throws IOException {
             } catch (IOException ioe) {
                 // ignore the exception
                 if (SSLLogger.isOn && SSLLogger.isOn("ssl")) {
-                    SSLLogger.warning("output stream close failed", ioe);
+                    SSLLogger.warning("output stream close failed. Debug info only. Exception details:", ioe);
```

Comment:
I may look at this bug report different.  It is a problem that the user does not understand the debug log properly.   Debug log is for debug information only, and the debug log level indicates the level of the message.

It looks like there is too much duplicated information.  A log message has already indicated that the message is debug information only.  Otherwise, exception should has been thrown in application level.  The adding of "Exception details:" adds unnecessary dependency of the exception logging format.

It may be fine to keep it unchanged, as if the users understand the logging message and logging levels. This kind of information normally means there is something that an application should take care of.  That why we use a warning level log, rather than a fine level log message.

It we really want an update, may be we could have a documentation enhancement instead.

Similar comments for other update.

Reply:
Sorry that I did not have my comment earlier, and while I was typing the comment the update was integrated.  Please just ignore this comment.